src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imaginginterface.py

```diff
@@ -68,44 +68,49 @@ def get_metadata(self) -> DeepDict:
         # Extract many scan image properties and attach them as dict in the description
         if self.image_metadata is not None:
             extracted_description = json.dumps(self.image_metadata)
-            two_photon_series_metadata.update(description=extracted_description)
+            two_photon_series_metadata.update(comments=extracted_description)
```

Comment:
I just looked at the file, and the comments look really weird. I see the intention to keep as much information as possible but this is too much:

<img width="586" alt="Screenshot 2024-01-26 at 12 47 11" src="https://github.com/catalystneuro/mousev1-to-nwb/assets/24475788/b0d17ecc-5b22-4d0f-b804-e1833f54a300">

I think a better way to store all this information is either to create an extension for this, if that is out of scope than add it as a DynamicTable. 

Let me know what you think

Reply:
I agree, but I also think it's not worth to keep all of this info. I think I will just remove from comments.